scanner/decisions/0005-scannerv4-read-cvss-data.md

```diff
@@ -0,0 +1,27 @@
+# 0005 - Adapt Scanner V4 NVD CVSS data updating pipeline to integrate with NVD CVE Api
+
+- **Author(s):** Yi Li <yli3@redhat.com>
+- **Created:** [2023-10-10]
+
+## Status
+
+Accepted
+
+## Context
+
+The National Vulnerability Database (NVD) has announced the retirement of its JSON feeds by December 15, 2023. This creates the need to change our existing CVSS data updater pipeline for Scanner V4, which currently relies on these JSON feeds.
+
+## Decision
+
+The current NVD CVSS GitHub Workflow will be updated to fetch data from the NVD CVE API instead of NVD Json feeds while maintaining the freshness of the NVD CVSS data bundle in Google Storage. It will categorize the downloaded data by CVSS V3 severity levels, and these categorized data will be archived into individual zip files, such as `severity-low.zip`. The format of the JSON is the same as before.
+
+The CVSS Updater in Central will be modified to download zip files corresponding to four distinct severity levels and write them into a single zip file stored in Central's file system. This eliminates the need for JSON parsing in the updater. ScannerV4 will pull the zip bundle and populate the Matcher DB with the enrichment data.
+
+## Consequences
+
+Transitioning from the NVD JSON feed to the NVD CVE API introduces certain constraints in data retrieval within the GitHub Action workflow, such as rate limiting and pagination. As a result, we will encounter a higher number of smaller, segmented JSON files.
```

Comment:
> Transitioning from the NVD JSON feed to the NVD CVE API introduces certain constraints in data retrieval within the GitHub Action workflow, such as rate limiting and pagination.

Oh, I see you have some concerns about the NVD API availability. I still think it is worth clarifying the differences otherwise we can really evaluate what the constraints you're referencing are.